selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/ActivityStream.java

```diff
@@ -1933,8 +1933,9 @@ public void openEditOnlineFromDocumentPreview (String document) {
     public static void deleteDocumentFromAS (String Document) {
         homePagePlatform.refreshUntil($(byText(Document)), exist, 500);
         String idActivity2 = $(byText(Document)).parent().parent().parent().parent().getAttribute("id").split("MediaName")[1].split("0")[0];
-        $(byId(ELEMENT_ACTIVITY_DROPDOWN.replace("{id}", idActivity2))).waitUntil(Condition.visible, openBrowserTimeoutMs).click();
-        $(byId(ELEMENT_DELETE_ACTIVITY_LINK.replace("{id}", idActivity2))).waitUntil(visible,openBrowserTimeoutMs).click();
+       refresh();
```

Comment:
Why do we need to refresh the page again ?

Reply:
We should understand what is the problem instead. Here we refresh the page until the document appears, then we refresh again. We should not have to refresh again.